lib/explorer/data_frame.ex

```diff
@@ -400,13 +400,14 @@ defmodule Explorer.DataFrame do
     * `:eol_delimiter` - A single character used to represent new lines. (default: `"\n"`)
   """
   @doc type: :io
-  @spec from_csv(filename :: String.t(), opts :: Keyword.t()) ::
+  @spec from_csv(entry :: String.t() | fs_entry(), opts :: Keyword.t()) ::
```

Comment:
Nitpick: I would still call it `filename` because that's the most obvious name. :) We should also document that it accepts `FSS` entries or a "s3://" style schema. However, to avoid repeating ourselves, maybe a section in the module docs under the IO bits?